src/main/java/Classes/Auto.java

```diff
@@ -1,5 +1,6 @@
 package Classes;
```

Comment:
```suggestion
package сlasses;
```

https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html